src/opm/parser/eclipse/EclipseState/Grid/FieldPropsManager.cpp

```diff
@@ -142,6 +143,70 @@ bool FieldPropsManager::tran_active(const std::string& keyword) const {
     return this->fp->tran_active(keyword);
 }
 
+template<class MapType>
+void apply_tran(const std::unordered_map<std::string, TranCalculator>& tran,
+                const MapType& double_data,
+                std::size_t active_size,
+                const std::string& keyword, std::vector<double>& data)
+{
+    const auto& calculator = tran.at(keyword);
+    for (const auto& action : calculator) {
+        const auto& action_data = double_data.at(action.field);
+
+        for (std::size_t index = 0; index < active_size; index++) {
+
+            if (action_data.value_status[index] != value::status::deck_value)
+                continue;
+
+            switch (action.op) {
+            case ScalarOperation::EQUAL:
+                data[index] = action_data.data[index];
+                break;
+
+            case ScalarOperation::MUL:
+                data[index] *= action_data.data[index];
+                break;
+
+            case ScalarOperation::MAX:
+                data[index] = action_data.data[index];
```

Comment:
What is `MAX` supposed to do?  If it is supposed to perform an action that resembles the `MAXVALUE` keyword, then this should be
```C++
data[index] = std::min(data[index], action_data.data[index]);
```

Reply:
Indeed, that went missing while moving to FieldData in opm-simulators. Thanks.